Common/BaselineSelection.py

```diff
@@ -302,6 +302,15 @@ def RecoJetSelection(df):
     #df.Define("JetBInclSize","Jet_idx[Jet_bIncl].size()").Define("JetbCandSize", "Jet_idx[Jet_bCand].size()").Display({"JetBInclSize", "JetbCandSize"}).Print()
     return df.Filter("Jet_idx[Jet_bCand].size()>=2 || FatJet_idx[FatJet_bbCand].size()>=1", "Reco bjet candidates")
 
+def AdditionalRecoJetSelection(df):
+    df = df.Define("AdditionalJet_B0", f"v_ops::pt(Jet_p4)>20 && abs(v_ops::eta(Jet_p4)) < 5 && ( Jet_jetId & 2 ) && (Jet_puId>0 || v_ops::pt(Jet_p4)>50)")
+    df = df.Define("AdditionalFatJet_B0", "FatJet_msoftdrop > 30 && abs(v_ops::eta(FatJet_p4)) < 5")
+    df = df.Define(f"AdditionalJet_B1", "RemoveOverlaps(Jet_p4, AdditionalJet_B0,{{httCand.leg_p4[0], httCand.leg_p4[1],HbbCandidate.leg_p4[0],HbbCandidate.leg_p4[1]},}, 2, 0.5)")
+    df = df.Define(f"AdditionalFatJet_B1", "RemoveOverlaps(FatJet_p4, AdditionalJet_B0,{{httCand.leg_p4[0], httCand.leg_p4[1],HbbCandidate.leg_p4[0],HbbCandidate.leg_p4[1]},}, 2, 0.5)")
```

Comment:
- don't remove overlaps between FatJet and b jet candidates
- call it SelectedFatJet, because "additional" is not a good description in this case

Reply:
ok